src/main/java/com/adobe/franklin/fragments/converter/Model.java

```diff
@@ -31,39 +32,34 @@ public SimpleSQLStatement toDropSQL() {
     }
     
     public SimpleSQLStatement toCreateSQL() {
-        StringBuilder buff = new StringBuilder();
-        buff.append("create table \"" + tableName + "\"(\n");
         // TODO probably the primary key needs to be a combination of the path and the variation
-        buff.append("    \"_path\" varchar(8000) primary key,\n");
-        buff.append("    \"_variation\" varchar(8000)");
+        StringBuilder buff = new StringBuilder()
+                .append("create table \"").append(tableName).append("\"(\n")
+                .append("    \"_path\" varchar(8000) primary key,\n")
+                .append("    \"_variation\" varchar(8000)");
         for (Column col : columns) {
-            buff.append(",\n");
-            buff.append("    " + col.toCreateSQL());
+            buff.append(",\n    ").append(col.toCreateSQL());
         }
         buff.append("\n)");
         return new SimpleSQLStatement(buff.toString());
     }
     
     private String toInsertStatement() {
         StringBuilder buff = new StringBuilder();
-        buff.append("insert into \"" + tableName + "\"(\"_path\", \"_variation\"");
+        buff.append("insert into \"").append(tableName).append("\"(\"_path\", \"_variation\"");
```

Comment:
I think it's this is not really needed. But it's not a problem either.

Reply:
I got irritated by the IntelliJ warning :b